src/plugins/intel_gpu/tests/unit/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -4877,6 +4877,94 @@ TEST(fully_connected_3d_onednn_gpu, compressed_int4_scale_static) {
         ASSERT_NEAR(output_ptr_ref[i], output_ptr[i], 9.0) << "i = " << i << std::endl;
     }
 }
+
```

Comment:
could you attach this to an existing test case? I guess we can reuse most of the code.